src/styles/themes/_default-theme.scss

```diff
@@ -807,11 +812,11 @@
 
   // ------ System UI Button theme ------
   --button-system-ui-text-color: var(--grey-tertiary-color);
-  --button-system-ui-background-color: var(--background-color);
+  --button-system-ui-background-color: transparent;
```

Comment:
this is going to have visual impact. For example:
<img width="529" alt="Screenshot 2023-06-28 at 11 35 05 PM" src="https://github.com/EightfoldAI/octuple/assets/106610186/b7317187-2370-4ae8-9076-9f58f3666aa2">
Are we comfortable with this?

Reply:
Why is transparent causing this visual? Shouldn't it use the background of the card?

Reply:
The above does not reflect the intended specification, the background color of the button at the top right should not be white.

Reply:
Will revert